### PR TITLE
Remove containers for non-existing projects

### PR DESCRIPTION
**Pull request by michael-molchanov**

### CODE REVIEW

File: bin/fin [hunk 1]
```diff
@@ -672,6 +672,22 @@ _remove_containers ()
 	fi
 }
 
+# Remove containers for removed projects.
+_remove_containers_removed_project_files ()
+{
+	echo-green "Removing containers for removed projects..."
```
Comment (achekulaev) [inline bin/fin]: Could you remove this message please and replace it with `check_docker_running`

**Comment by achekulaev** [comment]
Looks good to me

**Comment by lmakarov** [comment]
I would like to see a bats test for this use case prior to merging it in.

File: bin/fin [hunk 2]
```diff
@@ -672,6 +672,25 @@ _remove_containers ()
 	fi
 }
 
+# Remove containers for removed projects.
+_remove_containers_removed_project_files ()
```
Comment (lmakarov) [inline bin/fin]: This function name is not good and difficult to tell what it is about.
Let's change it to `_remove_dangling_containers`

File: bin/fin [hunk 3]
```diff
@@ -705,6 +724,7 @@ _docker_cleanup ()
 
 	_remove_dangling_images
 	_remove_dangling_volumes
+	_remove_containers_removed_project_files
```
Comment (lmakarov) [inline bin/fin]: Change to `_remove_dangling_containers`

**Pull request merged**